src/main.ts

```diff
@@ -9,6 +9,9 @@ async function bootstrap() {
     app.get<IConfigurationService>(IConfigurationService);
   const applicationPort: string =
     configurationService.getOrThrow('applicationPort');
+
+  app.enableCors();
```

Comment:
Why is this change necessary? I'd suggest we move it to a separate PR as it is unrelated.